### PR TITLE
change to Bond v4

### PR DESCRIPTION
**Pull request by yanamura**
update demo to use v4

### CODE REVIEW

File: BondDemo/ListViewController.swift [hunk 1]
```diff
@@ -24,19 +24,20 @@ class ListViewController: UITableViewController {
...
-      let cell = (self.tableView.dequeueReusableCellWithIdentifier("cell", forIndexPath: NSIndexPath(forItem: index, inSection: 0)) as? ListCellView)!
-      viewModel.name ->> cell.nameLabel
-      viewModel.username ->> cell.ownerLabel
-      viewModel.photo ->> cell.avatarImageView
+    dataSource.bindTo(tableView) { (indexPath, dataSource, tableView) -> UITableViewCell in
+      let cell = (self.tableView.dequeueReusableCellWithIdentifier("cell", forIndexPath: indexPath) as? ListCellView)!
+      let viewModel = dataSource[indexPath.section][indexPath.row]
+      viewModel.name.bindTo(cell.nameLabel.bnd_text)
+      viewModel.username.bindTo(cell.ownerLabel.bnd_text)
+      viewModel.photo.bindTo(cell.avatarImageView.bnd_image)
```
Comment (srdanrasic) [inline BondDemo/ListViewController.swift]: These bindings should be disposed in cell's `prepareForResue` method. The best way is to create a dispose bag as a property on cell, add the disposables returned by the `bindTo` method to that bag and dispose the bag on `prepareForResue`.

**Comment by srdanrasic** [comment]
Thank you @yanamura3! I have just one comment.

**Comment by yanamura** [comment]
@srdanrasic , Thanks for your review comment.
I fixed unbinding from cell when reused.

**Comment by ivanmoskalev** [comment]
Closes #4.

**Pull request merged**